parlai/agents/fairseq/fairseq.py

```diff
@@ -4,346 +4,412 @@
 # LICENSE file in the root directory of this source tree. An additional grant
 # of patent rights can be found in the PATENTS file in the same directory.
 
-from parlai.core.agents import Agent
 from parlai.core.dict import DictionaryAgent
 
 try:
-    from .fairseq_py.fairseq import models
+    from fairseq import models, optim
 except ImportError:
-    raise RuntimeError("Please run 'python setup.py build' and "
-                       "'python setup.py develop' from the fairseq_py directory")
-from .fairseq_py.fairseq.models import fconv
-from .fairseq_py.fairseq.multiprocessing_trainer import MultiprocessingTrainer
-from .fairseq_py.fairseq import criterions
-from .fairseq_py.fairseq import dictionary
-from .fairseq_py.fairseq.sequence_generator import SequenceGenerator
-from .fairseq_py.fairseq import options
-
-from torch.autograd import Variable
-
-from collections import deque
+    raise RuntimeError(
+        "Please run \"pip install 'git+https://github.com/pytorch/"
```

Comment:
This command doesn't do anything for me; it says `Requirement already satisfied: fairseq from git+https://github.com/pytorch/fairseq.git@v0.5.0#egg=fairseq in ./ParlAI/parlai/agents/fairseq/fairseq_py` - are there additional steps i need to take to get this running?

Reply:
You may have to `pip uninstall fairseq`, especially if you had done the `cd fairseq_py; python setup.py develop` previously.

Reply:
great that worked, thanks

Reply:
I should change it to `pip install --upgrade`